### PR TITLE
[fix](case) add order by in repositoryAffinityList1.sql

### PR DESCRIPTION
**Pull request by hello-stephen**
http://152.136.199.195:8115/buildConfiguration/DorisGit_DorisBranch20_NereidsAsan_DorisAllPLevelLoadData/15147?buildTab=tests&expandedTest=build%3A%28id%3A15147%29%2Cid%3A27377

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
run buildall

File: regression-test/suites/github_events_p2/sql/repositoryAffinityList1.sql [hunk 1]
```diff
@@ -9,5 +9,5 @@ WHERE (event_type = 'WatchEvent') AND (actor_login IN
     WHERE (event_type = 'WatchEvent') AND (repo_name IN ('ClickHouse/ClickHouse', 'yandex/ClickHouse'))
 )) AND (repo_name NOT IN ('ClickHouse/ClickHouse', 'yandex/ClickHouse'))
 GROUP BY repo_name
-ORDER BY stars DESC
+ORDER BY stars DESC, repo_name
```
Comment (yiguolei) [inline regression-test/suites/github_events_p2/sql/repositoryAffinityList1.sql:12]: group by stars, repo_name desc?

**Review by yiguolei — APPROVED** [review]
lgtm

**Review by mrhhsg — APPROVED** [review]
LGTM

**Pull request merged**